harvesting/oaipmh.py

```diff
@@ -118,9 +119,16 @@ def _get_redirect(software, headers):
 
 def _get_zenodo_identifier(redirect_url, headers):
     response = requests.head(redirect_url, headers=headers)
+    while rate_limit_reached(response):
+        # throttle requests
+        logger.info("Sleeping for 60 seconds to avoid HttpError 429 from {0}".format(redirect_url))
```

Comment:
Add something to the log message so we can make a distinction in which while loop (this or L140) we are sleeping.

Reply:
done